docs/codeql/reusables/supported-versions-compilers.rst

```diff
@@ -40,3 +41,4 @@
     .. [9] Requires glibc 2.17.
     .. [10] Support for the analysis of Swift requires macOS.
     .. [11] TypeScript analysis is performed by running the JavaScript extractor with TypeScript enabled. This is the default.
+    .. [12] Support for GitHub Actions is in public preview.
```

Comment:
Note: This will need updating for GA, but I agree to merge this initial version of the docs ASAP and iterate.

Reply:
Yup - will drop this footnote for GA.